stdlib/public/SDK/Foundation/URL.swift

```diff
@@ -289,14 +289,14 @@ public struct URLResourceValues {
     public var quarantineProperties: [String : Any]? {
         get {
             // If a caller has caused us to stash NSNull in the dictionary (via set), make sure to return nil instead of NSNull
-            if let isNull = _values[.quarantinePropertiesKey] as? NSNull {
+            if _values[.quarantinePropertiesKey] is NSNull {
                 return nil
             } else {
                 return _values[.quarantinePropertiesKey] as? [String : Any]
             }
         }
         set {
-            if let v = newValue {
+            if newValue != nil {
```

Comment:
This could be written as `_set(.quarantinePropertiesKey, newValue: newValue ?? NSNull())`. I've put that into [this PR](https://github.com/apple/swift/issues/6506) which is a superset of these plus some other warnings.